com.playeveryware.eos/Runtime/Core/Utility/FileSystemUtility.cs

```diff
@@ -507,10 +507,7 @@ public static async Task<string> ReadAllTextAsync(string path)
             // appropriate.
             try
             {
-                Debug.Log("Starting async file read");
-                var result = await File.ReadAllTextAsync(path);
-                Debug.Log("Finished async file read");
-                return result;
+                return await File.ReadAllTextAsync(path);
```

Comment:
This change should not be a part of this PR - since it was made in a different branch that has since been merged in. This might be something that will go away when you update the base branch from `development`. If it doesn't go away though, then this is curious.

Reply:
It went away after more rebasing.